### PR TITLE
ieee802154: add SFD and length of PHR for MR-FSK.

### PR DESCRIPTION
**Pull request by jeandudey**
### Contribution description
Specifies the SFD for MR-FSK on uncoded PHR + PSDU mode.

Screenshot from 2021-01-26 21-29-36

Also specifies the MR-FSK phy header (PHR) length in bytes. This is _will_ be used on #13295 .

### Testing procedure

- N/A

### Issues/PRs references

- Cherry-picked from #13295 , to not mix things up on that PR which is quite large.

### CODE REVIEW

File: sys/include/net/ieee802154.h [hunk 1]
```diff
@@ -120,6 +120,21 @@ extern "C" {
...
+/**
+ * For the MR-FSK PHY, the SFD value when PHR + PSDU are uncoded
+ * and with phyMRFSKSFD = 0 or 1 respectively
+ *
+ * 802.15.4g, Table 131 (p. 51)
+ *
+ *  @{
+ */
+#define IEEE802154_PHY_MR_FSK_2FSK_UNCODED_SFD_0 (0x90E4)
+#define IEEE802154_PHY_MR_FSK_2FSK_UNCODED_SFD_1 (0x7A0E)
```
Comment (benpicco) [inline sys/include/net/ieee802154.h:135]: Might also add the coded ones while you are at it.

**Review by jia200x — APPROVED** [review]
ACK

File: sys/include/net/ieee802154.h (hunk 1 above)
Comment (jeandudey) [reply sys/include/net/ieee802154.h]: Done, added the coded values.

**Review by jia200x — APPROVED** [review]
Re-ACK then ;)

**Comment by jeandudey** [comment]
Thanks for the review :-)

**Pull request merged**